apps/kitty/kitty.talon

```diff
@@ -0,0 +1,55 @@
+app.exe: kitty
+app.name: kitty
+-
+
+# Kitty is a terminal emulator for Unix systems (including WSL, unofficially).
+# Learn more at https://sw.kovidgoyal.net.
+
+tag(): terminal
+tag(): user.generic_unix_shell
+tag(): user.unix_utilities
+tag(): user.readline
+tag(): user.git
+
+# Cribbed from the .talon file for iTerm
+suspend: key(ctrl-z)
+resume:
+    insert("fg")
+    key(enter)
+resume [<user.number_string>]:
+    insert("fg " + number_string)
+    key(enter)
+
+# Overrides for basic editing, tab, and window management.
+# Note that if your `kitty_mod` is set to something other than the default of
+# ctrl-shift, you will need to make the same change here.
+# Note that some window managers (particularly GNOME3) may have conflicts with
+# some key combos
+
+copy that: key(ctrl-shift-c)
+(pace | paste) that: key(ctrl-shift-v)
+
+tab open: key(ctrl-shift-t)
```

Comment:
You should probably just utilize the `user.tabs` tag. If needed you can context override the actions.

Here you can read about actions
https://talon.wiki/Customization/Talon%20Framework/actions